data/xml/Q15.xml

```diff
@@ -10,46 +11,51 @@
-    </paper>
+  </paper>
 
   <paper id="1003">
     <title>Efficient Inference and Structured Learning for Semantic Role Labeling</title>
     <author><first>Oscar</first><last>Täckström</last></author>
     <author><first>Kuzman</first><last>Ganchev</last></author>
     <author><first>Dipanjan</first><last>Das</last></author>
     <year>2015</year>
-    <href>https://tacl2013.cs.columbia.edu/ojs/index.php/tacl/article/download/465/105</href>
+    <month>December</month>
+    <doi>10.1162/tacl_a_00120</doi>
+    <abstract>We present a dynamic programming algorithm for efficient constrained inference in semantic role labeling. The algorithm tractably captures a majority of the structural constraints examined by prior work in this area, which has resorted to either approximate methods or off-the-shelf integer linear programming solvers. In addition, it allows training a globally-normalized log-linear model with respect to constrained conditional likelihood. We show that the dynamic program is several times faster than an off-the-shelf integer linear programming solver, while reaching the same solution. Furthermore, we show that our structured model results in significant improvements over its local counterpart, achieving state-of-the-art results on both PropBank- and FrameNet-annotated corpora.</abstract>
     <pages>29–41</pages>
     <url>http://www.aclweb.org/anthology/Q15-1003</url>
-    <video href="http://techtalks.tv/talks/efficient-inference-and-structured-learning-for-semantic-role-labeling/61788/" tag="video"/>
-    <abstract>We present a dynamic programming algorithm for efficient constrained inference in semantic role labeling. The algorithm tractably captures a majority of the structural constraints examined by prior work in this area, which has resorted to either approximate methods or off-the-shelf integer linear programming solvers. In addition, it allows training a globally-normalized log-linear model with respect to constrained conditional likelihood. We show that the dynamic program is several times faster than an off-the-shelf integer linear programming solver, while reaching the same solution. Furthermore, we show that our structured model results in significant improvements over its local counterpart, achieving state-of-the-art results on both PropBank- and FrameNet-annotated corpora.</abstract>
-    </paper>
+    <video href="http://techtalks.tv/talks/efficient-inference-and-structured-learning-for-semantic-role-labeling/61788/" tag="video" />
+  </paper>
 
   <paper id="1004">
-    <title>SPRITE: Generalizing Topic Models with Structured Priors</title>
-    <author><first>Michael</first><last>Paul</last></author>
+    <title>S</title>
```

Comment:
Looks like a title error here.

Reply:
Also I wonder if you can wrap these in the `<fixed-case>` tag? We could also apply that in post-processing.

Reply:
What's the heuristic here for `<fixed-case>`?

Reply:
It’s more complicated than that (#123), so you could ignore this for now. Typically, you want to protect things like ALL-CAPS acronyms or proper nouns like `{E}nglish`. But we’re working on a pipeline to apply these later so let’s defer that piece.

Reply:
Ok, deferred. What's the title error?

Reply:
The title is just "S" :)

Reply:
I'm literate, I swear!

Reply:
I enjoyed this one.